forest/shared/src/lib.rs

```diff
@@ -31,16 +31,26 @@ pub fn db_path(config: &crate::cli::Config) -> PathBuf {
 #[cfg(feature = "rocksdb")]
 pub fn open_db(
     path: &std::path::Path,
-    config: &cli::Config,
+    config: Option<&cli::Config>,
```

Comment:
Let's use the default configuration instead of passing in `None` here.

Reply:
Done